machine/machine.go

```diff
@@ -69,6 +69,9 @@ func (e *EndpointStore) _updateMachines(
 			m.M.Host = ahost.Hostname
 			m.M.Active = true
 			m.M.Aws = e.config.GetAwsInfo(ahost.AwsMetadata)
+			if ahost.AwsMetadata != nil {
+				m.M.Region = ahost.AwsMetadata.Region
```

Comment:
I forgot that I haven't abstracted `Region` out of `AwsMetadata`. That's something I should do, but I want to consider how to do that to avoid potential conflicts between environments (i.e. AWS and GCP), where the same region name may be used.
Let's move ahead with this as-is, but note that we'll revisit this later.